proposals/p0722.md

```diff
@@ -22,7 +22,7 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
             -   [Full receiver type](#full-receiver-type)
             -   [Method names line up](#method-names-line-up)
             -   [Receiver in square brackets](#receiver-in-square-brackets)
-            -   [Receiver parameter is named `me`](#receiver-parameter-is-named-me)
+            -   [Receiver parameter is named `self`](#receiver-parameter-is-named-self)
```

Comment:
Maybe this should remain unchanged to reflect the history of the project?

Reply:
It should probably remain unchanged.

We generally only do important typo fixes or updating of links in proposals. While it's fine to try to keep *navigating* the proposal working (link updates), we shouldn't try to make them living documents. It's OK (and even good) for them to reflect the state of the language and project when they were accepted.